packages/governance/test/unitTests/test-committee.js

```diff
@@ -224,4 +225,12 @@ test('committee-open question:mixed', async t => {
 
   const questions = await publicFacet.getOpenQuestions();
   t.deepEqual(questions.length, 2);
+
+  const doc = {
+    node: 'committees.Economic_Committee',
+    owner: 'a committee contract',
+    pattern: 'mockChainStorageRoot.thisElectorate.',
+    replacement: 'published.committees.Economic_Committee.',
```

Comment:
these would read more easily as a map, even if it has just one item.
```suggestion
    rekey: {'mockChainStorageRoot.thisElectorate': published.committees.Economic_Committee`},
```